mne_qt_browser/tests/test_pg_specific.py

```diff
@@ -81,3 +82,276 @@ def test_annotations_interactions(raw_orig, pg_backend):
+    assert fig.mne.fig_settings is not None
+    assert pg_backend._get_n_figs() == 2
+    fig._fake_click_on_toolbar_action('Settings', wait_after=500)
+    assert pg_backend._get_n_figs() == 1
+    assert fig.mne.fig_settings is None
+    fig._fake_click_on_toolbar_action('Settings', wait_after=500)
+    assert pg_backend._get_n_figs() == 2
+    assert fig.mne.fig_settings is not None
+    fig._fake_click_on_toolbar_action('Settings', wait_after=500)
+    assert pg_backend._get_n_figs() == 1
+    assert fig.mne.fig_settings is None
+
+    fig._fake_click_on_toolbar_action('Settings', wait_after=500)
+    assert fig.mne.fig_settings is not None
+    downsampling_control = fig.mne.fig_settings.downsampling_box
+    assert downsampling_control.value() == fig.mne.downsampling
+
+    downsampling_control.setValue(2)
+    QTest.qWait(100)
+    assert downsampling_control.value() == 2
+    assert downsampling_control.value() == fig.mne.downsampling
+
+    downsampling_control.setValue(3)
+    QTest.qWait(100)
+    assert downsampling_control.value() == 3
+    assert downsampling_control.value() == fig.mne.downsampling
+
+    QTest.qWait(100)
+    downsampling_method_control = fig.mne.fig_settings.ds_method_cmbx
+    assert fig.mne.ds_method == downsampling_method_control.currentText()
+
+    downsampling_method_control.setCurrentText('mean')
+    QTest.qWait(100)
+    assert downsampling_method_control.currentText() == 'mean'
+    assert fig.mne.ds_method == 'mean'
+
+    downsampling_method_control.setCurrentText('subsample')
+    QTest.qWait(100)
+    assert downsampling_method_control.currentText() == 'subsample'
+    assert fig.mne.ds_method == 'subsample'
+
+    downsampling_method_control.setCurrentText('peak')
+    QTest.qWait(100)
+    assert downsampling_method_control.currentText() == 'peak'
+    assert fig.mne.ds_method == 'peak'
+
+    downsampling_method_control.setCurrentText('invalid_method_name')
+    QTest.qWait(100)
+    assert downsampling_method_control.currentText() != 'invalid_method_name'
+
+    sensitivity_control = fig.mne.fig_settings.scroll_sensitivity_slider
+    assert fig.mne.scroll_sensitivity == sensitivity_control.value()
+
+    sensitivity_control.setValue(100)
+    QTest.qWait(100)
+    assert sensitivity_control.value() == 100
+    assert fig.mne.scroll_sensitivity == 100
+
+    QTest.qWait(100)
+    sensitivity_values = list(range(sensitivity_control.minimum(),
+                                    sensitivity_control.maximum() + 1, 40))
+    if sensitivity_values[-1] != sensitivity_control.maximum():
+        sensitivity_values.append(sensitivity_control.maximum())
+    sensitivity_test = [False] * len(sensitivity_values)
+    for i in range(len(sensitivity_values)):
+        sensitivity_control.setValue(sensitivity_values[i])
+        QTest.qWait(50)
+        sensitivity_test[i] = \
+            fig.mne.scroll_sensitivity == sensitivity_control.value()
+    assert all(sensitivity_test)
```

Comment:
Do you agree this is simpler? Pytest can handle list comparisons directly, so:
```suggestion
    sensitivity_mne = list()
    sensitivity_qt = list()
    for val in sensitivity_values:
        sensitivity_control.setValue(val)
        QTest.qWait(50)
        sensitivity_mne.append(fig.mne.scroll_sensitivity)
        sensitivity_qt.append(sensitivity_control.value())
    assert sensitivity_mne == sensitivity_values
    assert sensitivity_qt == sensitivity_values
```